make/acceptance-tests.mk

```diff
@@ -28,10 +28,10 @@ test-acceptance: test-acceptance-setup ## Runs acceptance tests
 	$(PYTHON_VENV_DIR)/bin/behave --junit --junit-directory $(TEST_ACCEPTANCE_OUTPUT_DIR) --no-capture --no-capture-stderr $(TEST_ACCEPTANCE_TAGS_ARG) $(EXTRA_BEHAVE_ARGS) test/acceptance/features
 
 .PHONY: test-acceptance-x
-test-acceptance-x: test-acceptance-setup ## Runs acceptance tests for WIP tagged scenarios
+test-acceptance-x: test-acceptance-setup kustomize controller-gen opm envtest ## Runs acceptance tests for WIP tagged scenarios
```

Comment:
`envtest` shouldn't be needed, so long as we're not running unit tests as well.

Reply:
nice, updated